### DISCUSSION TITLE
Docs: How turbo repo handles affected packages with git integration?

### DISCUSSION THREAD

**Discussion by lucashfreitas** [vercel/turborepo#311] [Help] [answered]
How turborepo handles affected packages after git commits?

Sorry if I missed it, but I didn't find anything in docs talking about it, but I have found this article here on Nx mentioning that turborepo supports this: https://nx.dev/l/r/guides/turbo-and-nx?utm_source=twitter&utm_medium=ads&utm_campaign=dec2021.

Thanks

**Comment by teovillanueva** [comment]
Im having the same problem, this doesnt look like its documented properly

**Comment by lucashfreitas** [comment]
up

**Comment by weyert** [comment]
Don't  you mean the `--since` argument, see also: https://turborepo.org/docs/reference/command-line-reference#--since ?

**Comment by anthonyshew** [comment, marked as answer]
This is solved with the `--filter` flag. Docs can be found here: https://turbo.build/repo/docs/core-concepts/monorepos/filtering#filter-by-changed-workspaces

Note: weyert was correct at the time that `--since` did this - but that flag has been deprecated since then!
